### PR TITLE
YD-547 Fixed disappearing user photo link

### PR DESCRIPTION
**Pull request by Bert-R**
The UserDto class accidentally updated the user photo ID. This was wrong, as updating the user photo link is done differently.

### CODE REVIEW

File: appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy [hunk 1]
```diff
@@ -393,6 +393,36 @@ class UserPhotoTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
+	def 'Richard\'s photo link is still available after he updated his nickname'()
+	{
+		given:
+		User richard = addRichard()
+		def multipartEntity = MultipartEntityBuilder.create()
```
Comment (aristotelos) [inline appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy]: 🔧  Please use `uploadUserPhoto(richard)` below

File: core/src/main/java/nu/yona/server/subscriptions/service/UserDto.java [hunk 2]
```diff
@@ -189,7 +189,6 @@ void updateUser(User originalUserEntity)
 		originalUserEntity.setLastName(lastName);
 		originalUserEntity.setMobileNumber(mobileNumber);
 		originalUserEntity.setNickname(privateData.getNickname());
-		originalUserEntity.setUserPhotoId(privateData.getUserPhotoId());
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/subscriptions/service/UserDto.java]: Oops... I think this is a left-over from some earlier ideas on how to do user photo updates.

File: appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy (hunk 1 above)
Comment (Bert-R) [reply appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy]: Much better indeed. I didn't notice that helper method.

**Comment by Bert-R** [comment]
Pushed an update

**Pull request merged**